### DISCUSSION TITLE
What does this octopus emoji mean?

### DISCUSSION THREAD

**Discussion by SenaJp** [community/community#68329] [Programming Help] [answered]
I didn't find anything about it. Can you help me, what the hell does this octopus and these letters mean?

![image](https://github.com/community/community/assets/86388032/58151f58-8f54-455d-baca-7d18906080aa)
This appears in my terminal.

**Comment by ebubekirgungor** [comment, marked as answer]
The octopus is the mascot of Docker compose. The emoji shows that you are using docker-compose. The last characters are spaceship-prompt's default suffix. You can maybe change this in .env file like SPACESHIP_PACKAGE_SUFFIX="special suffix"
Documentation of spaceship-prompt: [https://spaceship-prompt.sh/sections/docker_compose/](https://spaceship-prompt.sh/sections/docker_compose/)
Spaceship prompt configuration: [https://spaceship-prompt.sh/config/prompt/](https://spaceship-prompt.sh/config/prompt/)

**Reply by SenaJp** [reply]
Thanks bro!

**Reply by ebubekirgungor** [reply]
You are welcome

**Reply by BradStell** [reply]
Was wondering wth that octopus was for years 🫠

Thanks!
